purchases/build.gradle

```diff
@@ -5,19 +5,16 @@ apply plugin: 'org.jetbrains.dokka'
 android {
     namespace 'com.revenuecat.purchases.api'
 
-// Commenting out flavor temporarily after main -> bc5-support merge. We will need to fix integration tests
-// after this.
-//    productFlavors {
-//        integrationTest {
-//            testApplicationId obtainTestApplicationId()
-//            testBuildType obtainTestBuildType()
-//            packagingOptions {
-//                resources.excludes.add("META-INF/*")
-//            }
-//            minSdkVersion 21 // Needed because mockk-android requires API 21
-//            matchingFallbacks = ['latestDependencies']
-//        }
-//    }
+    productFlavors {
+        integrationTest {
+            testApplicationId obtainTestApplicationId()
+            testBuildType obtainTestBuildType()
+            packagingOptions {
+                resources.excludes.add("META-INF/*")
+            }
+            minSdkVersion 21 // Needed because mockk-android requires API 21
```

Comment:
Note how these are running on android 21. Our previous integration tests (that still exist) run in android 16. I think it's worth to keep tests running in Android 16 to ensure compatibility, so I want to revisit this to see if we could use a lower version of mockk-android that supports android 16 without too many difficulties...